ogcapi-stable/ogcapi-features-core/src/main/java/de/ii/ldproxy/ogcapi/features/core/application/OgcApiFeaturesCoreConfiguration.java

```diff
@@ -34,6 +35,9 @@
     static final int MINIMUM_PAGE_SIZE = 1;
     static final int DEFAULT_PAGE_SIZE = 10;
     static final int MAX_PAGE_SIZE = 10000;
+    static final String PARAMETER_BBOX = "bbox";
+    static final String PARAMETER_DATETIME = "datetime";
+    static final String DATETIME_INTERVAL_SEPARATOR = "...";
```

Comment:
It is just an internal convention, so the exact string is not important, but in general "/" would be more intuitive as the slash is the separator in time intervals in ISO 8601.